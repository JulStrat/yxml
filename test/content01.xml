<a>ZOMFG! Element	content!

<entities>&amp;&lt;&gt;&apos;&quot;</entities>
<refs>&#x20;&#33;&#x0020;&#0033;</refs>
<![CDATA[CDATA!]]>
<![CDATA[[[CD<a/> <!-- no comment -->&amp;<?NotaPI?>&notaref;
]x]]y]]]z]]]]>
</a>
