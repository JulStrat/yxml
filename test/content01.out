
elemstart a
content
data ZOMFG! Element\x09content!\x0a\x0a
elemstart entities
content
data &<>'"
elemend
data \x0a
elemstart refs
content
data  ! !
elemend
data \x0aCDATA!\x0a[[CD<a/> <!-- no comment -->&amp;<?NotaPI?>&notaref;\x0a]x]]y]]]z]]\x0a
elemend
ok
